stacker/context.py

```diff
@@ -10,13 +10,17 @@ class Context(object):
 
     """
 
-    _optional_keys = ('environment', 'stack_names', 'parameters', 'mappings',
-                      'config')
-
-    def __init__(self, namespace, **kwargs):
+    def __init__(self, namespace, environment=None, stack_names=None,
```

Comment:
`_optional_keys` wasn't providing us with much, so I went ahead with just definining all the arguments and giving them reasonable defaults.  landscape.io will complain about the number of arguments, but not sure we want an options object for this.